fe/pom.xml

```diff
@@ -346,7 +347,7 @@ under the License.
                 <!-- for java-cup -->
                 <repository>
                     <id>cloudera-public</id>
-                    <url>https://repository.cloudera.com/artifactory/public/</url>
+                    <url>https://repository.cloudera.com/repository/libs-release-local/</url>
```

Comment:
I didn't find the corresponding document, but this address is indeed available. Could you provide some more detailed information? In addition, maybe the address we should use is `https://repository.cloudera.com/artifactory/cloudera-repos/`, FYI https://docs.cloudera.com/documentation/enterprise/release-notes/topics/cdh_vd_cdh5_maven_repo.html

Reply:
Problem i met:
```
sh build.sh --fe
```
prompted with
<img width="1663" alt="image" src="https://github.com/apache/doris/assets/42906151/cf05a595-120b-412a-9833-39a8f40b2530">
```
Plugin net. sourceforge.czt.dev:cup-maven-plugin:1.6-cdh or one of its dependencies could not be resolved: Failure to find net.sou
rceforge. czt.dev: cup-maven-plugin: jar:1.6-ch in https://repo.maven.apache.org/maven2
```
How I get the add above address:
1. [open](https://repository.cloudera.com/#browse/search=keyword%3Djava-cup:NX.coreui.model.Component-1:6254a685ac93787f112088315872dfad)
2. click 
<img width="974" alt="image" src="https://github.com/apache/doris/assets/42906151/2c6c1308-41f9-46e2-9fd3-50b09ad0114a">
3. get new address https://repository.cloudera.com/repository/libs-release-local/net/sourceforge/czt/dev/java-cup/0.11-a-czt02-cdh/java-cup-0.11-a-czt02-cdh.jar

Reply:
I met same problem when fetching java-cup and cup-maven-plugin, so above pic and txt and url maybe mixed with two situation.

Correct url for cup-maven-plugin:

https://repository.cloudera.com/repository/libs-release-local/net/sourceforge/czt/dev/cup-maven-plugin/1.6-cdh/cup-maven-plugin-1.6-cdh.jar

Reply:
Feel free to modify any in-correct part of pr, I don know if this is a permanent solution, as you have seen the official docs still using the previous url... BUT, after i changed to new url, my problem is solved.

Reply:
> I didn't find the corresponding document, but this address is indeed available. Could you provide some more detailed information? In addition, maybe the address we should use is `https://repository.cloudera.com/artifactory/cloudera-repos/`, FYI https://docs.cloudera.com/documentation/enterprise/release-notes/topics/cdh_vd_cdh5_maven_repo.html↳

I got you. Should we use this address (`https://repository.cloudera.com/artifactory/cloudera-repos/`)? It's widely used and I could find specific official recommendations.

The address you provided is still available, but I can't find specific instructions for changing it.